### PR TITLE
add workflow for disconnected upi with enabled secureboot on vsphere

### PR DESCRIPTION
**Pull request by jinyunma**
add workflow for disconnected upi with enabled secureboot on vsphere

### CODE REVIEW

File: ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-ref.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ ref:
   - namespace: test-credentials
     name: ci-route-53
     mount_path: /var/run/vault/vsphere-aws
+  env:
+  - name: SECURE_BOOT_ENABLED
+    default: "false"
```
Comment (jianlinliu) [inline ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-ref.yaml:22]: Maybe we can add a brief document for the SECURE_BOOT_ENABLED env

Comment (jinyunma) [reply ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-ref.yaml:22]: agree, updated to add documentation part.

**Comment by jinyunma** [comment]
/assign @rvanderp3 
Could you help to review and approve it when you have a moment? thanks.
job "rehearse-vsphere-discon" failed due to the disk was full on private registry server of VMC when mirroring image, previous one was passed.

**Pull request merged**